src/test/java/automation_workshop/GalleryTest.java

```diff
@@ -0,0 +1,88 @@
+package automation_workshop;
+
+import chunks.GalleryOverlay;
+
+import data.Constant;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.support.PageFactory;
+import pages.HomePageFS;
+
+import utils.CommonFuntions;
+
+public class GalleryTest extends CommonFuntions implements Constant {
+
+
+    @Before
+    public void before(){
+
+        createDriver(driverType.chrome);
+        driver.get(Constant.URL_FS);
+        driver.manage().window().maximize();
+    }
+
+    @Test
+    public void openFirstImage() throws Exception {
+
+        HomePageFS myhomepage = PageFactory.initElements(driver, HomePageFS.class);
+        GalleryOverlay mygalleryoverlay = PageFactory.initElements(driver, GalleryOverlay.class);
+
+        myhomepage.goGalleryOverlay();
+        Thread.sleep(2000);
```

Comment:
Avoid having wait steps on the tests. Please move this to the required method. Also lets add a constant for this, like int LONG_WAIT=2000;